website/docs/d/cloudCredential.html.markdown

```diff
@@ -0,0 +1,29 @@
+---
+layout: "rancher2"
+page_title: "Rancher2: rancher2_cloud_credential"
+sidebar_current: "docs-rancher2-datasource-cloud_credential"
+description: |-
+  Get information on a Rancher v2 Cloud Credentials.
+---
+
+# rancher2\_cloud\_credential
+
+Use this data source to retrieve information about a Rancher v2 setting.
+
+## Example Usage
+
+```
+data "rancher2_cloud_credential" "test" {
+    name = "test"
```

Comment:
Is the name property unique per user/access key scope?

Reply:
Yep, cloud-credential name is unique.